typescript/packages/plugins/ionic/src/ionic.plugin.ts

```diff
@@ -0,0 +1,20 @@
+import { type Chain, PluginBase } from "@goat-sdk/core";
+import { EVMWalletClient } from "@goat-sdk/wallet-evm";
+import { IonicService } from "./ionic.service";
+
+interface IonicPluginOptions {
+  chainId: number;
```

Comment:
You get the `chainId` from the `walletClient`, no need to pass it as an option here

Reply:
already taken care of :)